perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp

```diff
@@ -214,37 +214,67 @@ bool BicycleTracker::predict(const double dt, KalmanFilter & ekf) const
   return true;
 }
 
+/**
+ * @brief measurement update with ekf on receiving detected_object
+ *
+ * @param object : Detected object
+ * @return bool : measurement is updatable
+ */
 bool BicycleTracker::measureWithPose(
   const autoware_auto_perception_msgs::msg::DetectedObject & object)
 {
-  constexpr int dim_y = 2;  // pos x, pos y, depending on Pose output
-  // double measurement_yaw =
-  //   tier4_autoware_utils::normalizeRadian(tf2::getYaw(object.state.pose_covariance.pose.orientation));
-  // {
-  //   Eigen::MatrixXd X_t(ekf_params_.dim_x, 1);
-  //   ekf_.getX(X_t);
-  //   while (M_PI_2 <= X_t(IDX::YAW) - measurement_yaw) {
-  //     measurement_yaw = measurement_yaw + M_PI;
-  //   }
-  //   while (M_PI_2 <= measurement_yaw - X_t(IDX::YAW)) {
-  //     measurement_yaw = measurement_yaw - M_PI;
-  //   }
-  //   float theta = std::acos(
-  //     std::cos(X_t(IDX::YAW)) * std::cos(measurement_yaw) +
-  //     std::sin(X_t(IDX::YAW)) * std::sin(measurement_yaw));
-  //   if (tier4_autoware_utils::deg2rad(60) < std::fabs(theta)) return false;
-  // }
+  // yaw measurement
+  double measurement_yaw = tier4_autoware_utils::normalizeRadian(
+    tf2::getYaw(object.kinematics.pose_with_covariance.pose.orientation));
+
+  // prediction
+  Eigen::MatrixXd X_t(ekf_params_.dim_x, 1);
+  ekf_.getX(X_t);
+
+  // TODO(yoshiri): set better angle threshold for validation
+  /// angle threshold to filter
+  double yaw_threshold = M_PI_2;  // do not filter anything
+
+  // validate if orientation is available
+  bool use_orientation_information = false;
+  if (
+    object.kinematics.orientation_availability ==
+    autoware_auto_perception_msgs::msg::DetectedObjectKinematics::SIGN_UNKNOWN) {  // has 180 degree
+                                                                                   // uncertainty
+    // fix orientation
+    while (M_PI_2 <= X_t(IDX::YAW) - measurement_yaw) {
+      measurement_yaw = measurement_yaw + M_PI;
+    }
+    while (M_PI_2 <= measurement_yaw - X_t(IDX::YAW)) {
+      measurement_yaw = measurement_yaw - M_PI;
+    }
+    // check if valid angle measurement
+    if (std::abs(X_t(IDX::YAW) - measurement_yaw) < yaw_threshold) {
+      use_orientation_information = true;
+    }
```

Comment:
I think no need this

Reply:
I just want to filter measurement outlier in this part (currently yaw_threshold is 90 deg, so this does not filter anything). 
But it can be redundant statements, ok I'll remove them.

Reply:
@YoshiRi If you fix for this comment, pls let me know with commit id.

Reply:
@yukkysaito 
I removed this description in this commit.
https://github.com/autowarefoundation/autoware.universe/pull/2147/commits/2f4dd61cb8b3b7a198fc695a6d559e160e4f2156

Reply:
@YoshiRi Thank you for reply. I'll resolve this.